docs/pages/versions/v46.0.0/sdk/location.md

```diff
@@ -21,20 +22,31 @@ import SnackInline from '~/components/plugins/SnackInline';
 
 ### Android permissions
 
-- This module requires the permissions for approximate and exact device location. It also needs the foreground service permission to subscribe to location updates, while the app is in use. The `android.permission.ACCESS_COARSE_LOCATION`, `ACCESS_FINE_LOCATION`, and `FOREGROUND_SERVICE` permissions are automatically added.
-- In order to use background location features, you also must add the `android.permission.ACCESS_BACKGROUND_LOCATION` and [submit your app for review and request access to use the background location permission](https://support.google.com/googleplay/android-developer/answer/9799150?hl=en).
+The `expo-location` module requires permission for approximate (`ACCESS_COARSE_LOCATION`) and precise (`ACCESS_FINE_LOCATION`) device location in the `android.permission` property in **app.json**. It also needs foreground service permission (`FOREGROUND_SERVICE`) to subscribe to location updates while the app is in use. When you install this module in your Expo project, all of the three permissions are added automatically.
+
+To use background location features, you must add the `ACCESS_BACKGROUND_LOCATION` and [submit your app for review and request access to use the background location permission](https://support.google.com/googleplay/android-developer/answer/9799150?hl=en).
+
+<AndroidPermissions permissions={['ACCESS_COARSE_LOCATION', 'ACCESS_FINE_LOCATION', 'FOREGROUND_SERVICE', 'ACCESS_BACKGROUND_LOCATION']} />
+
+#### Excluding a permission
+
+When your Expo project doesn't benefit from having particular permission included, you can omit it. For example, if your application doesn't need access to the precise location, you can exclude the `ACCESS_FINE_LOCATION` permission.
```

Comment:
Same here, maybe good to warn that excluding required permissions can break functionality of the API